30_regexp_lab/relab.txt

```diff
@@ -2,16 +2,33 @@ Use grep to print only the lines from "input" for which:
 
 1. There is a binary number in the line
 
+$ grep -E -w [0,1]+ input
```

Comment:
1. Better to use `egrep`
2. No need to put a comma inside square brackets. Simply use `[01]+`